source/clear-linux/guides/maintenance/Telemetry-Enable-Disable.rst

```diff
@@ -0,0 +1,111 @@
+.. _telemetry-enable-disable:

```

Comment:
Keep file naming conventions. The file should be called telemetry-enable.rst and the label is then telemetry-enable
I don't think you have to put the enable and disable on the filename.

Reply:
What file naming conventions?